content/operate/kubernetes/deployment/openshift/openshift-operatorhub.md

```diff
@@ -66,11 +66,15 @@ You only need to install the SCC once, but you must not delete it.
 After the install, the OperatorHub automatically uses the constraint for Redis Enterprise node pods.
 
 {{< note >}}
-**Known Limitation** - The automatic use of the security constraint is limited. The
-Redis Enterprise must be named `rec` for the constraint to be used automatically.  **Use the cluster name `rec` when deploying with the OperatorHub.**
+If you are using the recommended RedisEnterpriseCluster name of `rec`, the SCC is automatically bound to the RedisEnterpriseCluster after install.
+
+If you choose a different name for the RedisEnterpriseCluster, or override the default service account name, you must manually bind SCC to the RedisEnterpriseCluster’s service account:
```

Comment:
```suggestion
If you choose a different name for the RedisEnterpriseCluster, or override the default service account name, you must manually bind the SCC to the RedisEnterpriseCluster’s service account:
```

Does this need a "the" since that's one specific SCC we're talking about?